docs/standard/clr.md

```diff
@@ -53,24 +53,9 @@ Language compilers and tools expose the runtime's functionality in ways that are
 - Garbage collection.
 
 - Use of delegates instead of function pointers for increased type safety and security. For more information about delegates, see [Common Type System](../../docs/standard/base-types/common-type-system.md).
+## CLR versions	
 
-## CLR versions
-
-The version number of the .NET Framework doesn't necessarily correspond to the version number of the CLR it includes. The following table shows how the two version numbers correlate:
-
-|.NET Framework version|Includes CLR version|
-|----------------------------|--------------------------|
-|1.0|1.0|
-|1.1|1.1|
-|2.0|2.0|
-|3.0|2.0|
-|3.5|2.0|
-|4|4|
-|4.5 (including 4.5.1 and 4.5.2)|4|
-|4.6 (including 4.6.1 and 4.6.2)|4|
-|4.7 (including 4.7.1 and 4.7.2)|4|
-|4.8|4|
-
+The .NET Framework version number doesn't necessarily correspond to the version number of the CLR it includes. For a list of .NET Framework versions and their corresponding CLR versions, see [.NET Framework versions and dependencies](../../framework/migration-guide/versions-and-dependencies.md). .NET Core releases have a single product version, that is, there is no separate CLR version. For a list of .NET Core versions, see [Download .NET Core](https://dotnet.microsoft.com/download/dotnet-core).
```

Comment:
The same .NET Framework link is also at the bottom in the "See also" section. Should we delete it from there?